src/Rx.KitchenSink.ts

```diff
@@ -39,8 +39,10 @@ import './add/observable/fromArray';
 import './add/observable/fromEvent';
 import './add/observable/fromEventPattern';
 import './add/observable/fromPromise';
+import './add/observable/if';
```

Comment:
@kwonoj in Rx4 `Observable.if` is in the "experimental" package, so I only put it into KitchenSink, though I'm still left wondering why we distinguish between "core" and "kitchen sink" operators in the first place.